ui/src/views/compute/EditVM.vue

```diff
@@ -289,15 +290,35 @@ export default {
       return decodedData.toString('utf-8')
     },
     fetchUserData () {
-      const params = {
-        id: this.resource.id,
-        userdata: true
+      let networkId
+      this.resource.nic.forEach(nic => {
+        if (nic.isdefault) {
+          networkId = nic.networkid
+        }
+      })
+      if (!networkId) {
+        return
+      }
+      const listNetworkParams = {
+        id: networkId
```

Comment:
Listall : true ?

Reply:
Hello @weizhouapache , I did not quite understand your question; do you want me to add a `listall: true` as a parameter in the request? Could you give me some details on why this is needed?

Reply:
Hi @FelipeM525 
if the caller (for example root admin) is different as the network owner (for example regular user), then `listall=true` is needed


see an example below
```
(testadmin) 🐱 > list networks id=69a230d7-1534-4cc9-b791-b98fdc2a845e filter=name
(testadmin) 🐱 > 
(testadmin) 🐱 > 
(testadmin) 🐱 > list networks id=69a230d7-1534-4cc9-b791-b98fdc2a845e filter=name listall=true
{
  "count": 1,
  "network": [
    {
      "name": "testadmin-003"
    }
  ]
}
```